01week/rockPaperScissors.js

```diff
@@ -9,6 +9,44 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  // there are two players, I'll call them hand1 and hand2
```

Comment:
you are repeating hand1.toLowerCase() for every condition. If you find yourself repeating, then you need to refactor. 
you can set a global variable to do that before hand to stop repeating. Ex: const player1 = hand1.toLowerCase(); then use player1 in your conditional statements